docs-src/descriptions/one-source.pug

```diff
@@ -829,8 +829,26 @@ pre(title='events in time').
                  12      14
 div
 
++descr-method('thru', 'thru', 'obs.thru(callback)').
+  This transformation method takes a single callback and calls the callback with
+  #[b obs] observable. Based on the most.js method of the same name, this
+  allows you to integrate your helper functions into chains of Kefir method calls
+  without adding them to Observable prototype. Functions that take and return
+  an observable can be passed to #[b thru], allowing the fluent interface to be
+  used consistently.
```

Comment:
Maybe rewrite it like this?

```
+descr-method('thru', 'thru', 'obs.thru(transformer)').
  Calls #[b transformer] with the #[b obs] as argument and returns whatever 
  transformer returned. `o.thru(fn)` is the same as `fn(o)`. This allows you to 
  integrate your helper functions into chains of Kefir method calls without adding 
  them to Observable prototype. Based on the most.js method of the same name.
```

I.e. get to the point more quickly and remove some redundancy.

Also I'm on the fence about mentioning that it was based on most.js method. I'm totally ok with giving them credit for the idea. But as a docs reader, this makes me think that in order to fully understand this method I need to make myself familiar with how it works in most.js, which is not the case.

Reply:
That's true; my thought initially was to actually link to the most.js API docs for the function, because they have a deeper explanation, but perhaps that's going overboard. If we're not going to do that, I'll remove the most.js reference.